Create new-supervisor-playbook.md

awichman opened the pull request:
@ccostino 👋🏻

awichman committed:
Create new-supervisor-playbook.md

Links to each play will be added to this intro page, as I create them.

github-actions[bot] committed:
format files

awichman committed:
Create play-1.md

github-actions[bot] committed:
format files

awichman committed:
Create play-2.md

github-actions[bot] committed:
format files

awichman committed:
Create play-3.md

github-actions[bot] committed:
format files

ccostino reviewed (commented):
Thank you so much, @awichman!  For the most part, everything looks good so far!  Two things jumped out:

* The spacing between list items - I noticed there was some in play 2 and that helped make it easier to read through the information, so I added suggestions for adding spacing between list items throughout all of the plays.
* Having the checkboxes be a part of the checklists doesn't seem to translate as well in displaying the Markdown on a webpage; I'm not sure how to address this though at the moment

I'll think on how we can solve the checkbox formatting!  If you agree with the spacing of list items, then that's the only thing I'd change here and we can keep going. :-)

awichman committed:
Update pages/supervisor-resources/new-supervisor-playbook/play-1.md

Co-authored-by: Carlo Costino <ccostino@users.noreply.github.com>

awichman committed:
Update pages/supervisor-resources/new-supervisor-playbook/play-1.md

Co-authored-by: Carlo Costino <ccostino@users.noreply.github.com>

awichman committed:
Update pages/supervisor-resources/new-supervisor-playbook/play-1.md

Co-authored-by: Carlo Costino <ccostino@users.noreply.github.com>

awichman committed:
Update pages/supervisor-resources/new-supervisor-playbook/play-2.md

Co-authored-by: Carlo Costino <ccostino@users.noreply.github.com>

awichman committed:
Update pages/supervisor-resources/new-supervisor-playbook/play-2.md

Co-authored-by: Carlo Costino <ccostino@users.noreply.github.com>

awichman committed:
Update pages/supervisor-resources/new-supervisor-playbook/play-3.md

Co-authored-by: Carlo Costino <ccostino@users.noreply.github.com>

github-actions[bot] committed:
format files

github-actions[bot] committed:
format files

github-actions[bot] committed:
format files

awichman committed:
Update pages/supervisor-resources/new-supervisor-playbook/play-3.md

Co-authored-by: Carlo Costino <ccostino@users.noreply.github.com>

awichman committed:
Update pages/supervisor-resources/new-supervisor-playbook/play-2.md

Co-authored-by: Carlo Costino <ccostino@users.noreply.github.com>

awichman committed:
Update pages/supervisor-resources/new-supervisor-playbook/play-2.md

Co-authored-by: Carlo Costino <ccostino@users.noreply.github.com>

github-actions[bot] committed:
format files

awichman committed:
Update pages/supervisor-resources/new-supervisor-playbook/play-3.md

Co-authored-by: Carlo Costino <ccostino@users.noreply.github.com>

github-actions[bot] committed:
format files

ccostino committed:
Removed checkbox formatting

We are finding that the checkbox formatting is not displaying as intended, so we are removing the checkboxes for the Handbook version.

github-actions[bot] committed:
format files

ccostino reviewed (approved):
Thank you so much for your help with this, @awichman!  We're good to go with this first batch. :-)
